test/testutils/testutilsloader.cpp

```diff
@@ -20,4 +19,11 @@ void setupLogger()
     OCC::Logger::instance()->setLogDebug(true);
 }
 Q_COREAPP_STARTUP_FUNCTION(setupLogger);
+
+void setUpAccountFactory()
```

Comment:
Hmm I assume we better rename setupLogger I wouldn't bet on the order the startup functions are called and when we have multiple, stuff might happen.

Reply:
I checked the code. The functions are added to a queue which is called in order, apparently. But anyway, the order doesn't matter at all, since all of these functions are idempotent.

Reply:
Still there is no reason to have more than one

Reply:
Sure, cleanup is on the way.